core/mm/core_mmu.c

```diff
@@ -642,6 +643,13 @@ uint32_t core_mmu_type_to_attr(enum teecore_memtypes t)
 	case MEM_AREA_NSEC_SHM:
 		return attr | TEE_MATTR_PRW | cached;
 	case MEM_AREA_EXT_DT:
+	/*
```

Comment:
This should probably be indented one more step.

Reply:
Done